src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -246,9 +268,9 @@
     @Argument(
             fullName  = FuncotatorArgumentDefinitions.TRANSCRIPT_LIST_LONG_NAME,
             optional = true,
-            doc = "Set of transcript IDs to use for annotation to override selected transcript."
+            doc = "File to use as a list of transcripts (one transcript ID per line) OR A set of transcript IDs to use for annotation to override selected transcript."
```

Comment:
Do the transcript IDs need to have the version numbers (e.g. ENST00000123456.5) or not (e.g. ENST00000123456)?

Reply:
Version numbers are ignored - I'll update the documentation.